python/ray/serve/context.py

```diff
@@ -34,7 +34,6 @@ class ReplicaContext:
     app_name: str
 
 
-@PublicAPI(stability="alpha")
```

Comment:
Just for my own understanding, what does it meant when a public api doesn't have a stability tag? Is it stable and final? Or is it equivalent to alpha?

Reply:
this is not a public API (I don't know why it was labeled, it's never been documented/recommended)

we should not have any public APIs that aren't annotated